### ISSUE TITLE
Tests for discretizing methods using ContinuousFactor 

### ISSUE THREAD

**Issue by ankurankan** [pgmpy/pgmpy#732]
Currently tests have been written using `ContinuousNode`.

**Comment by kris-singh** [comment]
@ankurankan can't find these tests.

**Comment by yashu-seth** [comment]
@kris-singh This will have to wait. We don't have working discretize methods for ```ContinuousFactor```.

**Comment by kris-singh** [comment]
@yashu-seth yes i saw that. Though i think the univariate case works fine. I also submitted the PR for #742 
that solves discretised the for the univariate Continous Factor
